tests/visual/percy-storybook/Css.percy.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import DataTable from 'dash-table/dash/DataTable';
+
+const setProps = () => { };
+
+storiesOf('DashTable/CSS override', module)
+    .add('css with escaped id', () => (<DataTable
+        setProps={setProps}
+        id={'`~!@#$%^&*()=+ \\|/.,:;\'"`?[]<>{}'}
```

Comment:
Without the fix, the table does not render and throws.

Reply:
leading to the developer screaming "~!@#$%^&*()=+ \\|/.,:;\'"`?[]<>{}"